### PR TITLE
feat: add option to control scm font size

### LINKED ISSUE #114945: Improve font options for SCM input
Hello!

It seems there's no way to control `font-size` and `font-weight` for the SCM input widget, despite there being an option for `font-family`. When using my main editor font, this results in uncomfortably large text which is far from ideal. This also may be considered an accessibility issue since users who would prefer a certain font can't adjust it to maximize legibility.

If possible, I'd like to see these options added. Perhaps they could optionally mirror the editor settings similar to how `scm.inputFontFamily` does as well.

### PR DESCRIPTION
**Pull request by krishnaanaril**
This PR fixes #114945

### CODE REVIEW

File: src/vs/workbench/contrib/scm/browser/scm.contribution.ts [hunk 1]
```diff
@@ -198,6 +198,13 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).regis
 			markdownDescription: localize('inputFontFamily', "Controls the font for the input message. Use `default` for the workbench user interface font family, `editor` for the `#editor.fontFamily#`'s value, or a custom font family."),
 			default: 'default'
 		},
+		'scm.inputFontSize': {
+			type: 'number',
+			minimum: 6,
```
Comment (krishnaanaril) [inline src/vs/workbench/contrib/scm/browser/scm.contribution.ts:203]: Hi @eamodio, The values here are kinda magic numbers, Can we move these constants to somewhere else?

File: src/vs/workbench/contrib/scm/browser/scm.contribution.ts [hunk 2]
```diff
@@ -198,6 +198,13 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).regis
...
+			maximum: 100,
+			markdownDescription: localize('inputFontSize', "Controls the font size for the input message in pixels."),
+			default: 13
```
Comment (krishnaanaril) [inline src/vs/workbench/contrib/scm/browser/scm.contribution.ts:206]: I took the default value from the previous [configuration](https://github.com/microsoft/vscode/blob/480b74c02e59ad9103a5598df0ec39d9e010b875/src/vs/workbench/contrib/scm/browser/scmViewPane.ts#L1480).

**Comment by krishnaanaril** [comment]
Since this is a new feature, where do we add tests? I didn't find any scm font related test case. May be I've missed.

**Comment by krishnaanaril** [comment]
@eamodio  Seems you are bit busy these days. :wink:

**Comment by gjsjohnmurray** [comment]
@eamodio any reason not to merge this in time for 1.55? Looks good to me.

File: src/vs/workbench/contrib/scm/browser/scm.contribution.ts (hunk 1 above)
Comment (eamodio) [reply src/vs/workbench/contrib/scm/browser/scm.contribution.ts:203]: Do we need to specify a min/max? Looking at our other (non-editor font sizes) we don't have any limits specified.

**Comment by eamodio** [comment]
Thanks for the contribution!

**Comment by krishnaanaril** [comment]
@gjsjohnmurray @eamodio Thank you. This was my first contribution to VS Code 😄

**Pull request merged**